docs/howto/openstack/nova/new-server.md

```diff
@@ -23,90 +23,114 @@ to work with the OpenStack CLI, please do not forget to [source the RC
-    _Create a Server_ will slide over. At the top, type in a name for the
-    new server and select one of the available regions.
+    You will notice several rounded boxes on that pane, each for defining,
+    configuring, and instantiating a different {{company}} Cloud object. Go
+    ahead and click the _Server_ box. A new pane titled _Create a Server_
+    will slide over. At the top, type in a name for the new server and
+    select one of the available regions.
 
     ![Create new server](assets/new-server/shot-02.png)
 
-    In the _Boot source_ section below, click the dropdown menu on
-    the left and make sure you select _Boot from image_, so you can choose
-    one of the readily available OS images to boot the new server off of.
-    To pick one of the images, click on the dropdown menu on the right. For
-    this how-to guide, we have chosen _ubuntu_ in general and
-    _Ubuntu 22.04 Jammy Jellyfish 20220810_ in particular.
-
-    ![Boot source](assets/new-server/shot-03.png)
-
-    Next, make sure _Boot Target_ is set to _Volume (Recommended)_.
-    Regarding the server's CPU core count and amount of memory, set the
-    [_Flavor_](../../../reference/flavors/index.md) accordingly.
-    The default flavor specifies a server with 1 CPU core and 1GB of RAM.
-    You can start with that or select a different configuration by clicking
-    the dropdown menu at the right of _Flavor_. Please note that, depending on
-    the chosen flavor, the estimated monthly cost of the server changes. (While a
-    server is shut off you are still getting charged for it, but less so.)
-    At any time, this estimated cost is displayed in the green rectangular area
-    at the top. Something else that affects the cost is the size of the root
-    device. Take a look at the _Size_ parameter below, and notice the
-    default (in
+    Below the region selection is a drop-down menu where you must select a
+    _server profile_. Currently, four profiles are available: _Generic_,
+    _High CPU_, _Low Latency Disk_, and _GPU_. Selecting each of those gives
+    you access to a subset of all available
+    [_flavors_](../../../reference/flavors/index.md). Now, as the profile
+    names suggest, each profile but the _Generic_ points to a set of flavors
+    suitable for servers intended to run specific types of applications.
+    More specifically, the _Generic_ profile is for general-purpose cloud
+    servers, the _CPU_ profile is for servers designed to host CPU-intensive
+    applications, the _Low Latency Disk_ is for servers that boot off of a
+    local low-latency disk, whereas the _GPU_ profile is for servers that
+    boot off of a local SSD disk and, most importantly, have direct access
+    to an NVIDIA vGPU. We should point out that _not_ all profiles may be
+    available in any region. Let us choose the _Generic_ profile for now,
+    which, by the way, is available in any of the regions.
```

Comment:
This wall of text is difficult to read when rendered. For better legibility, I would suggest to use bullet points for enumerating the server profiles. Also, use the imperative mood, and edit out fluff like "by the way".

```suggestion
    Below the region selection is a drop-down menu where you must select a
    _server profile_. Currently, four profiles are available:
    
    *  _Generic_,
    * _High CPU_,
    * _Low Latency Disk_, and
    *  _GPU_.
    
    Selecting one of those gives
    you access to a subset of all available
    [_flavors_](../../../reference/flavors/index.md).
    
    Now, as the profile
    names suggest, each profile but the _Generic_ points to a set of flavors
    suitable for servers intended to run specific types of applications:
    
    * The _Generic_ profile is for general-purpose cloud
    servers.
    * The _CPU_ profile is for servers designed to host CPU-intensive
    applications.
    * The _Low Latency Disk_ profile is for servers that boot off of a
    local low-latency disk.
    * The _GPU_ profile is for servers that
    boot off of a local SSD disk and, most importantly, have direct access
    to an NVIDIA vGPU. 
    
    We should point out that _not_ all profiles may be
    available in any region. Choose the _Generic_ profile for now,
    which, is available in any of the regions.
```